packages/svelte-form/tests/simple.test.ts

```diff
@@ -0,0 +1,74 @@
+/// <reference lib="dom" />
+import { afterEach, beforeEach, describe, expect, it } from 'vitest'
+import '@testing-library/jest-dom'
+import { userEvent } from '@testing-library/user-event'
+import { mount, unmount } from 'svelte'
+// @ts-ignore tsc doesn't know about named exports from svelte files
```

Comment:
i'd just separate this out, it isn't really anything to do with the component. rather it is input

so it could go in its own `sample-data.ts` or whatever you prefer to call it. then import it here and in `simple.svelte`

Reply:
Now that we use svelte-check to type-check this we can just remove the ts-ignore comment
```suggestion
```